README.md

```diff
@@ -346,3 +346,253 @@ python -m pytest -s -v
 ``` 
 
 from the cloudos-py main folder and 7 tests should pass. 
+
+## CohortBrowser worked example
+
+### Load the library
+
+Navigate to the root folder of the cloudos-py package and run cloudos-py from here. Initially, we will be importing the two main classes. These are Cohort (used to manage cohorts) and CohortBrowser (used to analyse a cohort). There are two other classes (Query and PhenoFilter) which are used for making a query of phenotypes in CloudOs and we will introduce them later.
+
+``` python
+from cloudos.cohorts import Cohort, CohortBrowser
+```
+
+### Configure CloudOS
+
+This package is primarily a means of communicating with a CloudOS instance using its API. This is done through the CohortBrowser and Cohort classes. Before it can  communicate with the CloudOS instance, the package must be configured with some key information:
+- The CloudOS base URL. This is the URL in your browser when you navigate to the Cohort Browser in CloudOS. Often of the form `https://my_instance.lifebit.ai/app/cohort-browser`.
+- The CloudOS token. Navigate to the settings page in CloudOS to generate an API key you can use as your token (see image below).
+- The CloudOS team ID. Also found in the settings page in CloudOS labelled as the "Workspace ID" (see image below).
+
+![CloudOS settings page](man/figures/settings_page.png)
+
+This information is given as environmental variables:
+
+```python
+>>> apikey = '***'
+>>> workspace_id = '***'
+>>> cloudos_url = "***"
+```
+To make a CohortBrowser object:
+
+```python
+>>> cb = CohortBrowser(apikey, cloudos_url, workspace_id)
+```
+
+### List Cohorts
+
+To check list of available cohorts in a workspace use `list_cohorts()` from the `CohortBrowser` class. This will
+provide a dictionary of cohorts. By default 10 are shown this can be varied with `size`. To get all cohorts use `size=all`.
```

Comment:
```suggestion
provide a dictionary of cohorts. By default 10 are shown this can be varied with the `size` argument. To get all cohorts use `size=all`.
```